app/apps/server/bridges/livechat.js

```diff
@@ -123,7 +124,7 @@ export class AppLivechatBridge {
 		return Livechat.transfer(
 			this.orch.getConverters().get('rooms').convertAppRoom(currentRoom),
 			this.orch.getConverters().get('visitors').convertAppVisitor(visitor),
-			{ userId: targetAgent.id, departmentId }
+			{ userId: targetAgent.id, department: LivechatDepartment.findOneById(departmentId) },
```

Comment:
You don't need to read the model here, you can do it inside the `Livechat.transfer`.